src/lib/util/Colors.js

```diff
@@ -74,7 +74,7 @@ class Colors {
 	/**
 	 * Apply the background
 	 * @since 0.5.0
-	 * @param {ColorsFormatType} background The background to apply
+	 * @param {?string} background The background to apply
```

Comment:
JSDocs doesn't use `?` to denote optional
```suggestion
	 * @param {string} [background] The background to apply
```